### PR TITLE
fixed tests for nova live migration

### PR DESCRIPTION
**Pull request by TregubovYury**
Tuning for nova live migration tests http://cz7776.bud.mirantis.net:8080/jenkins/job/9.0-NOVA_tests_generated_from_template/14/testReport/junit/mos_tests.nova.nova_test/NovaIntegrationTests/test_network_connectivity_to_v_m_during_live_migration__542823__/ and http://cz7776.bud.mirantis.net:8080/jenkins/job/9.0-NOVA_tests_generated_from_template/14/testReport/junit/mos_tests.nova.nova_test/NovaIntegrationTests/test_live_migration_of_v_ms_with_data_on_root_and_ephemeral_disk__542824__/

### CODE REVIEW

File: mos_tests/nova/nova_test.py [hunk 1]
```diff
@@ -550,3 +502,45 @@ def test_live_migration_of_v_ms_with_data_on_root_and_ephemeral_disk(self):
...
+        # Start ping of the vm in background
+        ping = subprocess.Popen(["/bin/ping", "-c20", "-i1", ip_to_ping],
+                                stdout=subprocess.PIPE)
+        # Then run the migration
+        self.nova.servers.live_migrate(instance, new_hyper,
+                                       block_migration=True,
+                                       disk_over_commit=False)
+        # Check that migration is over, usually it takes about 10-15 seconds
+        end_time = time() + 60 * timeout
+        while getattr(instance, "OS-EXT-SRV-ATTR:hypervisor_hostname") != \
```
Comment (gdyuldin) [inline mos_tests/nova/nova_test.py:520]: `common.functions.wait` ?

Comment (TregubovYury) [inline mos_tests/nova/nova_test.py:520]: Inherited from legacy code, have no time to change and test other solutions today.

**Comment by TimurNurlygayanov** [comment]
Need to fix comments.

**Comment by gdyuldin** [comment]
I've used this patch here - https://github.com/Mirantis/mos-integration-tests/pull/236/commits/6ae54aaa2adc7921b0e3016d6d5425b150da4119
This PR can be close after https://github.com/Mirantis/mos-integration-tests/pull/236 will be merged

**Comment by TregubovYury** [comment]
The code is merged to master in scope of the https://github.com/Mirantis/mos-integration-tests/pull/236

**Pull request closed without merging**